lms/static/sass/views/_verification.scss

```diff
@@ -821,6 +829,114 @@
 
     // indiv slides - review
     #wrapper-review {
+      color: $black;
+
+      .page-title {
+          border-bottom: 2px solid $darkGrey;
+          padding-bottom: 15px;
+          margin-bottom: 20px;
+          font-weight: bold;
+          text-transform: inherit;
+        }
+
+      .review {
+
+      .certificate {
+        font-size: 18px;
+        background-repeat: no-repeat;
+	    padding-left: 50px;
+	    overflow: hidden;
+	    min-height: 32px;
+
+          p {
+	        margin-top: 0;
+            line-height: 22px;
+            font-weight: 600;
+            color: $black;
+          }
+
+          .purchase {
+            float: right;
+            margin-left: 15px;
+            text-align: right;
+
+            .product-info {
+              color: #0079BC;
+              font-size: 22px;
+              font-weight: 600;
+            }
+        }
+
+        &.verified_icon {
+          background-image: url('#{$static-path}/images/icon-sm-verified.png');
+        }
+
+        &.no-id-professional_icon, &.professional_icon {
+          background-image: url('#{$static-path}/images/icon-sm-professional.png');
+        }
+      }
+
+      .payment-buttons {
+        overflow: auto;
+        margin-bottom: 15px;
+        margin-top: 10px;
+        padding-bottom: 5px;
+
+        .payment-button {
+          padding: 8px 20px;
+          min-width: 200px;
+          &.is-selected {
+            background: #0079BC !important;
+          }
+        }
+      }
+
+      hr {
+        border-width: 2px;
+        margin: 1.4em 0;
+      }
+    }
+
+      .container {
+        padding: 15px 0;
+
+        p {
+          line-height: 22px;
+          color: $black;
+        }
+        .photo-requirement {
+          position: relative;
+          padding-left: 40px;
+          margin-top: 15px;
+          font-size: 12px;
+          background-repeat: no-repeat;
+          background-position: left top;
+
+            h6 {
+              font-weight: bold;
+              color: #444444;
+            }
+
+          &:before {
+            content: " ";
+            position: absolute;
+            left: 0;
+            top: 2px;
+            font-family: 'FontAwesome';
+            font-size: 20px;
+            color: #9B9B9B;
+          }
+          &.cam_icon:before {
```

Comment:
NIT: hmm.. instead of using the unicode designation we should be able to use the existing classes for font awesome. (icon fa fa-nameoficon) should do the trick.